apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -506,3 +484,39 @@ const sendOptions = {
     backgroundColor: '#fff',
   },
 }
+
+const rampOnOffTransitionSpec = {
```

Comment:
while rebasing I think this commit was skipped / or wrongly merged into the changes https://github.com/Emurgo/yoroi/commit/14aa4a76703041dd24859ad1687997472aa4002e

Reply:
cc: @euvini @SorinC6